consensus/istanbul/backend/engine.go

```diff
@@ -419,20 +421,19 @@ func (sb *Backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 
 func (sb *Backend) getValSet(header *types.Header, state *state.StateDB) ([]common.Address, error) {
 	var newValSet []common.Address
-	validatorsAddress, err := sb.regAdd.GetRegisteredAddressAtStateAndHeader(params.ValidatorsRegistryId, state, header)
-	if err == core.ErrSmartContractNotDeployed {
+	// Get the new epoch's validator set
+	maxGasForGetValidators := uint64(1000000)
```

Comment:
Agreed! However, I'd prefer to implement this in a follow-on PR in order to avoid delaying this merge. I made an issue for this #362 and assigned myself. That work with you?